vector/api/endpoints/query-data.mdx

```diff
@@ -44,6 +44,23 @@ of fields below.
 <ParamField body="filter" type="string" default="">
   [Metadata filter](/vector/features/filtering) to apply.
 </ParamField>
+<ParamField body="weightingStrategy" type="string">
+  For sparse vectors of sparse and hybrid indexes, specifies what kind of
+  weighting strategy should be used while querying the matching non-zero
+  dimension values of the query vector with the documents.
+
+  If not provided, no weighting will be used.
+
+  Only possible value is `IDF` (inverse document frequency).
+</ParamField>
+<ParamField body="fusionAlgorithm" type="string">
+  Fusion algorithm to use while fusing scores
+  from dense and sparse components of a hybrid index.
+
+  If not provided, defaults to `RRF` (Reciprocal Rank Fusion).
+
+  Other possible value is `DBSF` (Distribution-Based Score Fusion).
+</ParamField>
```

Comment:
If there is a suggested documentation regarding weighting strategies or fusion algorithms, should we give reference to them?